src/main.rs

```diff
@@ -1,3 +1,5 @@
+#[macro_use]
+extern crate lazy_static;
```

Comment:
Prefer `std::sync`'s `Once, `OnceLock` for the next time

Reply:
Ah, I've already rid of of globals, this is an artifact

You were right :P, something was going wrong with them, and I've still no clue what